pkg/watchdogd/watchdogd.go

```diff
@@ -7,40 +7,63 @@
 //
 // It starts in the running+armed state:
 //
-//             \| watchdogd Running     | watchdogd Stopped
+//              | watchdogd Running     | watchdogd Stopped
 //     ---------+-----------------------+--------------------------
 //     Watchdog | watchdogd is actively | machine will soon reboot
 //     Armed    | keeping machine alive |
 //     ---------+-----------------------+--------------------------
 //     Watchdog | a hang will not       | a hang will not reboot
 //     Disarmed | reboot the machine    | the machine
 //
-// The following signals control changing state:
-//
-//     - STOP: running -> stopped
-//     - CONT: stopped -> running
-//     - USR1: armed -> disarmed
-//     - USR2: disarmed -> armed
+
 package watchdogd
 
 import (
 	"context"
+	"errors"
 	"fmt"
-	"io/ioutil"
 	"log"
+	"net"
 	"os"
-	"os/signal"
-	"path/filepath"
-	"strconv"
 	"strings"
 	"time"
 
 	"github.com/u-root/u-root/pkg/watchdog"
 	"golang.org/x/sys/unix"
 )
 
-// DaemonProcess is the name of the daemon process.
-const DaemonProcess = "watchdogd"
+// UDS is the name of daemon's unix domain socket.
+const UDS = "/tmp/watchdogd"
+
+const (
+	OpStop     = iota // 0 - Stop the watchdogd petting.
```

Comment:
it's better if these are printable and 0 and 1 are best avoided. 
Could you make it
OpStop='S'
OpContinue='C'
OpDisarm='D'
OpArm='A'

this makes watching the data go by a lot easier!

Reply:
Better idea! -- Done

I was a little paranoid about that string takes more bytes ("Stop", "Continue"), so took int instead, printing int directly does not make any sense without reading the code. I was thinking about maintaining a int to readable string map, and pretty print it to console, but that takes a few lines of code, and is not aesthetically clean.

single char is def better than digits, but single byte.